packages/protocol/contracts/common/test/MockToken.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.5.13;
```

Comment:
Nit: add ERC20 to the name so it's easier to look up.

Reply:
turns out there's already `MockERC20Token` so I just got rid of this file entirely